lib/ZumoOta/src/FileManager.cpp

```diff
@@ -80,3 +81,32 @@ bool FileManager::init()
     }
     return true;
 }
+
+size_t FileManager::read128Bytes(const char* firmwareName, uint8_t* buffer)

```

Comment:
Thats a rather strange convetion to have a readxXbytes function. It is unflexible for no gain.  User readBytes( len ...) instead and call it with 128 from the flash prog.

Reply:
It will be adjusted in the next commit.